app/models/feature_flag.rb

```diff
@@ -24,6 +24,13 @@ class FeatureFlag
       "Allow starting an application on this service directly after " \
         "completing an eligibility check.",
       "Thomas Leese"
+    ],
+    [
+      :test_user,
```

Comment:
```suggestion
      :staff_test_user_auth,
```

Just to make it slightly more explicit what this does.